client/react/frontpage/components/StreamBar.js

```diff
@@ -140,25 +152,25 @@ const StreamBar = React.createClass({
 Recently played tracks collapsable slider
 @prop expanded: should show recently played tracks
 */
-const RecentlyPlayed = React.createClass({
-  getInitialState() {
+var RecentlyPlayed = React.createClass({
+  getInitialState: function() {
```

Comment:
What's the difference between declaring a class method as:
```javascript
getInitialState() {
```
vs 
```javascript
getInitialState: function() {
```

I haven't seen the latter notation before!

Reply:
tbh same idk where any of these changes came from cuz i didn't change any of these lines for the stream marquee but ima change them back!!!